src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -2018,7 +2018,7 @@ class ChipDeviceController(ChipDeviceControllerBase):
     '''
 
     def __init__(self, opCredsContext: ctypes.c_void_p, fabricId: int, nodeId: int, adminVendorId: int, catTags: typing.List[int] = [
-    ], paaTrustStorePath: str = "", useTestCommissioner: bool = False, fabricAdmin: typing.Optional[FabricAdmin.FabricAdmin] = None, name: str = '', keypair: typing.Optional[p256keypair.P256Keypair] = None):
+    ], paaTrustStorePath: str = "", dacRevocationSetPath: str = "",  useTestCommissioner: bool = False, fabricAdmin: typing.Optional[FabricAdmin.FabricAdmin] = None, name: str = '', keypair: typing.Optional[p256keypair.P256Keypair] = None):
         super().__init__(
```

Comment:
My approval is with the assumption:
* Ordering comment above it addressed 
* https://github.com/project-chip/connectedhomeip/pull/37134 lands first so people can replicate your test

Reply:
Thanks @tehampson, #37134 is landed and I have addressed the reviews.